eox_nelp/user_profile/api/v1/urls.py

```diff
@@ -0,0 +1,10 @@
+"""eox_nelp puser_profile.api v1 urls"""
+from django.urls import path
+
+from eox_nelp.user_profile.api.v1.views import update_user_data
+
+app_name = "eox_nelp"  # pylint: disable=invalid-name
+
+urlpatterns = [
+    path("update-user-data/", update_user_data, name="update-user-data"),
```

Comment:
it could be
```suggestion
    path("user-data/", update_user_data, name="update-user-data"),
```
Then with HTTP verbs control the action to create, update, delete or is a very different think?

Reply:
Since the unique allowed verb is post the name helps to identify the view purpose